.gitignore

```diff
@@ -1,3 +1,3 @@
-/config-tool/SAMLBridgeConfiguration/SAMLBridgeConfiguration.suo
+*.suo
```

Comment:
You're going to want to add /PrecompiledWeb/ here as well, if this is where the solution will drop the files as before.